src/Components/MapComponent/MapComponent.jsx

```diff
@@ -98,13 +100,36 @@ class MapComponent extends PureComponent {
     this.projection = 'EPSG:3857';
 
     const dataLayer = new MapboxLayer({
-      name: 'Basemap',
+      name: 'data',
       visible: true,
-      isBaseLayer: true,
       url: `https://maps.geops.io/styles/base_bright_v2/style.json?key=${APIKey}`,
     });
 
+    const baseLayerOthers = new MapboxStyleLayer({
+      name: 'basemap.others',
+      mapboxLayer: dataLayer,
+      isBaseLayer: true,
+      visible: false,
+      styleLayersFilter: styleLayer => {
+        return /perimeter_mask_routing_europe$/.test(styleLayer.id);
+      },
+    });
+
+    const baseLayerFoot = new MapboxStyleLayer({
+      name: 'basemap.foot',
+      mapboxLayer: dataLayer,
+      isBaseLayer: true,
+      visible: false,
+      styleLayersFilter: styleLayer => {
+        return /perimeter_mask_routing_dach$/.test(styleLayer.id);
```

Comment:
Sorry I just realize you should use a metadata instead of the id.

Reply:
Done